docs/contributing.rst

```diff
@@ -165,6 +165,58 @@ subset of the test suite. In order to run a specific test run:
 
 The entire test suite is automatically run for each pull request.
 
+Performance measurement
+-----------------------
+
+To track the performance of various functionality of Modin, we use Asv_ tool.
+
+Various scenarios in which it may be needed with appropriate command:
+
+* It is necessary to check the impact of the new patch on the performance of a certain set of operations:
+
+.. code-block:: bash
+
+  asv continuous -f 1.05 src/master HEAD -b TimeGroupBy --launch-method=spawn
+
+* It is necessary to check the healthy of benchmarks after making changes or writing new ones:
```

Comment:
Maybe something like robustness or availability instead of healthy?

Reply:
I prefer using "presence of error" phrase.